packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -231,18 +248,7 @@ class VifAllowedIps extends BaseComponent {
         ) : (
           map(this._getIps(), (ip, ipIndex) => (
             <Row>
-              <Col size={10}>
```

Comment:
IMHO, this change is not related to this PR.

Reply:
It's related because trying to edit a "custom" IP wouldn't work with `SelectIp`. So I'm removing it since it's not very useful anyway.